packages/Toast/src/Toast.js

```diff
@@ -125,7 +128,7 @@ function Toast(props) {
       hasCloseButton={hasCloseButton}
       isFixed={isFixed}
       role="alert"
-      style={{ zIndex }}
+      style={{ zIndex: zIndex !== undefined ? zIndex : deafultZIndex }}
```

Comment:
Can we take this opportunity to migrate this inline style into the `styles.js`, please?